Gemfile.lock

```diff
@@ -180,10 +204,16 @@ GEM
       railties (>= 5.0)
     websocket-driver (0.7.4)
       websocket-extensions (>= 0.1.0)
+    websocket-driver (0.7.4-java)
+      websocket-extensions (>= 0.1.0)
     websocket-extensions (0.1.5)
 
 PLATFORMS
+  java
```

Comment:
did all of these gems come from a bundle update?

Reply:
Everything but shoulda matchers. We added those to do our model tests.